internal/daemon/daemon.go

```diff
@@ -827,9 +827,9 @@ func getListener(socketPath string, listenerMap map[string]net.Listener) (net.Li
 	}
 
 	runtime.LockOSThread()
-	oldmask := syscall.Umask(0111)
+	oldmask := unix.Umask(0111)
 	listener, err := net.ListenUnix("unix", address)
-	syscall.Umask(oldmask)
+	unix.Umask(oldmask)
```

Comment:
The change from syscall to unix, could you just add some insight for me why this is done ?

Reply:
Documentation states the `syscall` package is deprecated! https://pkg.go.dev/syscall